requirements.yaml

```diff
@@ -1,11 +1,5 @@
 ---
 # Add Ansible Galaxy Packages here, role_example_hello included to prevent errors in template testing
-roles:
-- irixjp.role_example_hello           # https://galaxy.ansible.com/irixjp/role_example_hello
```

Comment:
We should keep the hello world example, as it's a safe measure for debugging ansible

Reply:
What value does this role provide users building the container that they wouldn't already get by running the playbook and checking the results?

Reply:
If there's an issue building with the playbook, it'd be good to know whether it's an ansible issue or an issue specific to the thing we're installing. The hello example would at least tell us ansible seems to be working fine (or not)

Reply:
The traceback would tell us why the failure happened, right? I think it would be pretty easy to tell if it's failing from an internal Ansible issue versus a bad play in the playbook. That being said, I can understand your perspective on wanting to include this role. I guess I'm not completely opposed to putting it back in the image, but I want to make sure we _absolutely need it_ before I do.

Reply:
Per our conversation earlier, fe7257d readds the hello world role but on the condition that it's commented out until it's needed.